dev/fattest.simplicity/bnd.bnd

```diff
@@ -95,4 +97,8 @@ test.project: true
 	org.jboss.arquillian.core:arquillian-core-spi;version=1.3.0.Final,\
 	org.jboss.arquillian.core:arquillian-core-api;version=1.3.0.Final,\
 	org.jboss.arquillian.test:arquillian-test-spi;version=1.3.0.Final,\
+	com.ibm.ws.org.osgi.annotation.versioning;version=latest,\
+	org.testcontainers:database-commons;version=1.15.0,\
+	org.testcontainers:jdbc;version=1.15.0,\
```

Comment:
Update test containers to 1.15.1 and ducktape to 1.0.8

Reply:
I updated them.  I had to merge everything so it was necessary.  I also updated the other places that still had 1.15.0 and 1.0.7